docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
```

Comment:
Mounting the entire current directory into the container can lead to potential issues, such as overwriting files or including unnecessary files. Consider specifying only the necessary directories or files to be mounted.